src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -786,4 +786,10 @@ Valid from:</comment>
   <data name="NuGetLicense_LicenseElementMissingValue" xml:space="preserve">
     <value>The license element value is empty. This is likely due to an authoring error. </value>
   </data>
+  <data name="Error_RepositorySettings_UnsignedPackage" xml:space="preserve">
+    <value>The repository announced that all their packages where signed and the package was found unsigned.</value>
```

Comment:
The repository announced that all their packages where signed and the package was found unsigned. to
The repository announced that all their packages **are** signed but the package {0} is not.

Do we have information about the package id here? 
We should add it. 
If there's a wrapper over this error that specifies that, simply removing `{0}` should be enough. 

Happy to discuss these message offline.

Reply:
This will be wrapped in another message containing package id, version and source. So the full message will be: 

```
Package '{0} {1}' from source '{2}': The repository announced that all their packages are signed and the package is unsigned.
```

Reply:
Sounds good.